### PR TITLE
Fix - import OpenCV only if `verifier` is in CONCENT_FEATURES

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/550

### CODE REVIEW

File: concent_api/verifier/tasks.py [hunk 1]
```diff
@@ -36,6 +35,12 @@
 from .utils import unpack_archive
 
 
+# OpenCV is imported conditionally because it requires system module to work,
+# which is not installed on Concent instances without verifier.
+if 'verifier' in settings.CONCENT_FEATURES:
```
Comment (dybi) [inline concent_api/verifier/tasks.py]: isn't local importing the better approach?

Comment (rwrzesien) [reply concent_api/verifier/tasks.py]: Could be, the only advantage of current solution which I see is that import remains on top of the file. But generally removing `if` is also an advantage. So as you wish, I don't have any strong opinion on this one.

Comment (cameel) [reply concent_api/verifier/tasks.py]: I think it's good enough as is. I don't have a strong preference either way but having it on top of the file is indeed a bit more informative - you can see at a glance that the module is using OpenCV.

File: concent_api/verifier/tasks.py [hunk 2]
```diff
@@ -36,6 +35,12 @@
 from .utils import unpack_archive
 
 
+# OpenCV is imported conditionally because it requires system module to work,
```
Comment (cameel) [inline concent_api/verifier/tasks.py]: `system module` -> `a system package` or `OpenCV package installed in the system`

To make it clear that you don't refer to the `sys` module or any other Python module.

**Comment by bartoszbetka** [comment]
The problem still occurs. 
```
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/srv/http/virtualenv/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/srv/http/virtualenv/lib/python3.6/site-packages/django/core/management/__init__.py", line 338, in execute
    django.setup()
  File "/srv/http/virtualenv/lib/python3.6/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/srv/http/virtualenv/lib/python3.6/site-packages/django/apps/registry.py", line 116, in populate
    app_config.ready()
  File "/srv/http/concent_api/concent_api/apps.py", line 8, in ready
    from concent_api import system_check  # noqa, flake8 F401 issue  # pylint: disable=unused-variable
  File "/srv/http/concent_api/concent_api/system_check.py", line 13, in <module>
    from concent_api.constants  import AVAILABLE_CONCENT_FEATURES
  File "/srv/http/concent_api/concent_api/constants.py", line 6, in <module>
    import core.urls
  File "/srv/http/concent_api/core/urls.py", line 3, in <module>
    from .views import send, receive, receive_out_of_band, protocol_constants
  File "/srv/http/concent_api/core/views.py", line 8, in <module>
    from core.message_handlers          import handle_message
  File "/srv/http/concent_api/core/message_handlers.py", line 32, in <module>
    from core.queue_operations import send_blender_verification_request
  File "/srv/http/concent_api/core/queue_operations.py", line 1, in <module>
    from conductor.tasks import blender_verification_request
  File "/srv/http/concent_api/conductor/tasks.py", line 11, in <module>
    from verifier.tasks import blender_verification_order
  File "/srv/http/concent_api/verifier/tasks.py", line 41, in <module>
    import cv2
  File "/srv/http/virtualenv/lib/python3.6/site-packages/cv2/__init__.py", line 3, in <module>
    from .cv2 import *
ImportError: libSM.so.6: cannot open shared object file: No such file or directory
```

**Comment by kbeker** [comment]
Branch reverted

**Pull request merged**